specification/cognitiveservices/data-plane/Language/stable/2022-05-01/analyzetext-authoring.json

```diff
@@ -0,0 +1,2965 @@
+            "$ref": "#/definitions/TextAnalysisAuthoringExportedClass"
+          }
+        },
+        "documents": {
+          "description": "The list of documents belonging to this project.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TextAnalysisAuthoringExportedCustomSingleLabelClassificationDocument"
+          }
+        }
+      },
+      "x-ms-discriminator-value": "customSingleLabelClassification",
+      "x-ms-client-name": "ExportedCustomSingleLabelClassificationProjectAssets"
+    },
+    "TextAnalysisAuthoringExportedDocumentClass": {
+      "description": "Represents a classification label for a document.",
+      "type": "object",
+      "properties": {
+        "category": {
+          "type": "string"
+        }
+      },
+      "x-ms-client-name": "ExportedDocumentClass"
+    },
+    "TextAnalysisAuthoringExportedDocumentEntityLabel": {
+      "description": "Represents an entity label for a document.",
+      "type": "object",
+      "properties": {
+        "category": {
+          "description": "The entity category.",
+          "type": "string"
+        },
+        "offset": {
+          "format": "int32",
+          "description": "Start position for the entity text.",
+          "type": "integer"
+        },
+        "length": {
+          "format": "int32",
+          "description": "Length for the entity text.",
+          "type": "integer"
+        }
+      },
+      "x-ms-client-name": "ExportedDocumentEntityLabel"
+    },
+    "TextAnalysisAuthoringExportedDocumentRegion": {
+      "description": "Represents a region in a document for entity labeling.",
+      "type": "object",
+      "properties": {
+        "regionOffset": {
+          "format": "int32",
+          "description": "Start position for the region.",
+          "type": "integer"
+        },
+        "regionLength": {
+          "format": "int32",
+          "description": "Length for the region text.",
+          "type": "integer"
+        },
+        "labels": {
+          "description": "The list of entity labels belonging to this region.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TextAnalysisAuthoringExportedDocumentEntityLabel"
+          }
+        }
+      },
+      "x-ms-client-name": "ExportedDocumentRegion"
+    },
+    "TextAnalysisAuthoringExportedEntity": {
+      "description": "Represents an entity in an exported project.",
+      "type": "object",
+      "properties": {
+        "category": {
+          "description": "The entity category.",
+          "type": "string"
+        }
+      },
+      "x-ms-client-name": "ExportedEntity"
+    },
+    "TextAnalysisAuthoringExportedProject": {
+      "description": "Represents an exported project.",
+      "required": [
+        "project-file-version",
+        "stringIndexType",
+        "metadata"
+      ],
+      "type": "object",
+      "properties": {
+        "project-file-version": {
```

Comment:
Property names should be lower camel case, e.g. "projectFileVersion". Kebab case is problematic in languages like JavaScript.